### ISSUE TITLE
Bug: db connection on android doesn't support ForeignKey

### ISSUE THREAD

**Issue by antailyaqwer** [tamimattafi/kabin#5]
As said in [documentation](https://www.sqlite.org/foreignkeys.html#fk_enable), connection should enable foreign key support in runtime.
Please, add configuration for this or make this behavior enabled by default

**Comment by tamimattafi** [comment]
According to `SQLDelight` docs, Foreign support should be enabled during driver creation
https://cashapp.github.io/sqldelight/2.0.1/multiplatform_sqlite/foreign_keys/

**Comment by tamimattafi** [comment]
Added new flags and callbacks to `KabinDatabaseConfiguration` in [db530a3](https://github.com/tamimattafi/kabin/commit/db530a3485071c3a0cdbf994c8703da2775c0748)

Here's how you control ForeignKey behaviour:
```kotlin 
val configuration = KabinDatabaseConfiguration(
    context = this,
    name = "sample-database",
    foreignKeyConstraintsEnabled = false
)
```
`foreignKeyConstraintsEnabled` is `true` by default

**Comment by tamimattafi** [comment]
Changes were published to [0.1.0-alpha03](https://github.com/tamimattafi/kabin/releases/tag/0.1.0-alpha03)
